src/test/java/seedu/address/testutil/SessionBuilder.java

```diff
@@ -61,13 +59,9 @@ public SessionBuilder withExerciseType(String exerciseType) {
     /**
      * Sets the {@code Interval} of the {@code Session} that we are building.
      */
-    public SessionBuilder withInterval(String start, String duration) {
-        try {
-            this.interval = new Interval(SessionParserUtil.parseStringToDateTime(start), Integer.parseInt(duration));
-            return this;
-        } catch (ParseException e) {
-            throw new RuntimeException(e);
-        }
+    public SessionBuilder withInterval(LocalDateTime start, int duration) {
```

Comment:
just curious, any reason why this takes in LocalDateTime instead of String (since the other with__ takes in string)?

Reply:
hmm, the other time when I asked for a workaround in the group, it was regarding this issue but I think changing it to String works to actually! Anyway in the tests, I've added the conversions to LocalDateTime and int respectively, so it works as well.